src/cpp/multisolver_interface/SolverConfig.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include <algorithm>
 #include <utility>
+#include "antares-xpansion/xpansion_interfaces/LogUtils.h"
```

Comment:
Useless

Reply:
It is not. It defines the type of exceptions used in the file